examples/css2d_label.html

```diff
@@ -133,11 +135,8 @@
 			function animate() {
 
 				requestAnimationFrame( animate );
-
-				var elapsed = clock.getElapsedTime();
-
-				moon.position.set( Math.sin( elapsed ) * 5, 0, Math.cos( elapsed ) * 5 );
-
+				moon.position.applyAxisAngle( moonDir, Math.PI / 120 );
+				earth.rotateOnAxis( earthDir, Math.PI / 120 );
```

Comment:
css2d_label.html is just a behavior example and I think this change makes it slightly more complex. Could we keep this as it is?

Reply:
sure